### PR TITLE
[FEAT] Add 'X-XSS-Protection: 1; mode=block' header by default

### PR DESCRIPTION
**Pull request by madmatt**
[FEAT] Add 'X-XSS-Protection: 1; mode=block' header by default to all CWP projects, with option to disable
[DOCS] Add documentation for the configurable values (xss_protection_disabled, egress_proxy_default_enabled and egress_proxy_exclude_domains)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,35 @@
...
+
+You can disable the egress proxy entirely by adding the following YML configuration:
+```
+InitialisationMiddleware:
+    egress_proxy_default_enabled: false
+```
+
+You can also add to the list of domains to disable the proxy by adding the following YML configuration:
+```
+InitialisationMiddleware:
```
Comment (robbieaverill) [inline README.md]: Looks great! Can we just switch this for `CWP\Core\Control\InitialisationMiddleware` and use two spaces for YAML indentation?

Comment (madmatt) [reply README.md]: Ah yes, of course! I'm still thinking like an SS3 dev ;-)

**Comment by madmatt** [comment]
Have made the requested changes, squashed and force-pushed now @robbieaverill :)

**Comment by robbieaverill** [comment]
LGTM!

File: README.md [hunk 2]
```diff
@@ -4,6 +4,35 @@
...
+This module includes core configuration that is required for Common Web Platform sites to function correctly.
+
+### Configuration
+There are some settings that can be modified in this module, depending on requirements. These are listed below.
+
+#### XSS Protection
+By default, CWP sites instruct newer browsers to protect against cross-site scripting (XSS) attacks. This is done using an HTTP header (X-XSS-Protection). More information on this header can be found on the [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection) site. To disable this feature, add the following to your YML configuration:
+```
+CWP\Core\Control\InitialisationMiddleware:
+  xss_protection_disabled: true
```
Comment (chillu) [inline README.md]: Not a big fan of double negatives - it should be `xss_protection_enabled: false`

**Pull request merged**